src/types/Firestore.ts

```diff
@@ -84,5 +85,6 @@ export interface Firestore {
     allUsers?: Array<WithId<User>>;
     userModalVisits?: Array<WithId<UserVisit>>;
     chatRequests?: Array<WithId<ChatRequest>>;
+    chatUsers?: Array<WithId<User>>;
```

Comment:
This is more for my own knowledge than anything, but is there any difference between `Array<WithId<User>>` and `WithId<User>[]` (or similar) in TypeScript?

Reply:
No difference apparently: https://stackoverflow.com/questions/36842158/arraytype-vs-type-in-typescript

Reply:
TIL! :)